### ISSUE TITLE
Foundation 6.5 do not work with Jasmine tests

### ISSUE THREAD

**Issue by mariodd** [foundation/foundation-sites#11929]
## Expected Behavior
Tests written in [Jasmine](https://jasmine.github.io/) with foundation-sites js loaded (it doesn't even need to be inizialized) should work.

## Current Behavior
Tests are run twice and errors are present in console.

## Possible Solution
Consider `triggers` loading, since a quick debug directs the attention to `foundation.util.triggers.js` and related [methods](https://github.com/foundation/foundation-sites/commit/13d9ca20576dc614e5a4c686f153079bbed7c95a#diff-7f0f7b3a3c16aa6b7716a29b1a814775R76) .

## Test Case and/or Steps to Reproduce (for bugs)
https://codepen.io/mariodd/pen/BaaeVzX

How to reproduce:
1. Load Foundation (in codepen.io I left only the conflicting files for convenience) and Jasmine js
2. Write and run any test

## Your Environment
- Foundation version(s) used: from Foundation 6.5.3, but bug started with 6.5.0
- Browser(s) name and version(s): any
- Device, Operating System and version: any

## Checklist
- [x] I have read and follow the CONTRIBUTING.md document.
- [x] This is a bug report or a feature request.
- [x] There are no other issues similar to this one.
- [x] The issue title and template are correctly filled.

**Comment by DanielRuf** [comment]
Hi @mariodd, try waiting until everything is fully loaded:

https://codepen.io/DanielRuf/pen/XWJXVmX

**Comment by DanielRuf** [comment]
Changing the order seems to fix this:

https://codepen.io/DanielRuf/pen/vYELpWr

Seems we have to bisect this.

**Comment by DanielRuf** [comment]
12822b44785108aa3be923fcb4365ae77c0d3fc6 introduced this issue.

**Comment by DanielRuf** [comment]
Not sure about the triggers here. We should probably revert this to

```js
    if(document.readyState === "complete") {
      Triggers.Initializers.addSimpleListeners();
      Triggers.Initializers.addGlobalListeners();
    } else {
      $(window).one('load', () => {
        Triggers.Initializers.addSimpleListeners();
        Triggers.Initializers.addGlobalListeners();
      });
    }
```

@joeworkman what do you and the others think?

**Comment by mariodd** [comment]
Hello, @DanielRuf any update on this?

**Comment by DanielRuf** [comment]
> Hello, @DanielRuf any update on this?

No, as you can see there is no update so far. PRs and further input are very welcome.
